crates/sui-rest-api/openapi/openapi.json

```diff
@@ -5502,6 +5510,146 @@
           }
         }
       },
+      "UnresolvedGasPayment": {
+        "type": "object",
+        "required": [
+          "objects",
```

Comment:
does this need to be required?

I don't love an empty list being treated as an indicator that coin selection should happen

Reply:
oh yes nice catch, it doesn't need to be required i'll make that change